roadmap.md

```diff
@@ -31,48 +31,61 @@ be refined if appropriate.
 * P0. Stable API for Remote execution including platform description
 * P0. List of feature to deprecate until version 1.0 are tracked in a publicly available document
 * P1. Bazel on Windows does not need to install MSYS
-* P2. Bazel can load workspace recursively
+* P1. Migration tool: Maven project to Bazel project
 
 #### 0.7
 2017-09
 
 * P0. Skylark is fully documented; strategy for user-provided Skylark rule documentation
-* P0. Support for Android integration testing
-* P0. Support for Robolectric test for Android
+* P0. Support for Android integration testing (on Linux)
+* P0. Support for Robolectric test for Android (on Linux)
 * P1. The Build Event Protocol is stable
 * P1. Support for coverage can be extended in Skylark and C++
 * P1. Support for testing Skylark rules
+* P1. iOS test runner
+* P1. Better CLI output (--experimental-ui is enabled by default)
 * P2. All external repositories can use the local cache
 * P2. Local caching of build artifacts
```

Comment:
twerth is signed up for the experimental ui. I don't know who's working on iOS, if anyone.